workspace/scm.py

```diff
@@ -193,6 +193,15 @@ def merge_branch(branch, squash=False, strategy=None):
         cmd.append('--squash')
     if strategy:
         cmd.append('--strategy=' + strategy)
+
+    current = current_branch()  # uses current_branch() method  to get current branch
+    message = ""
+    if current:
+        message = "Merge branch %s into %s" % (branch, current)
```

Comment:
use .format() to construct strings

Reply:
Better yet, use format strings: `f"Merge branch {branch} into {current}"`